### PR TITLE
Add support for llama-3.1-8b-instruct and qwen-2-7b

### LINKED ISSUE #80: [v0.1.x] (Functionality) CacheGen support for different models
# Description

Currently, CacheGen only supports a few models. 

CacheGen will perform best if its configuration is tuned for each model. But we should also have a "default fallback" for un-tuned models. 

This may require a refactor of the current CacheGen configuration code

# How to test

This should be covered by some end-to-end tests. When passing some CacheGen-untuned models, it should still be able to reduce TTFT compared to using safetensor compression

### PR DESCRIPTION
**Pull request by YaoJiayi**
This request resolves #80 #135 .
`Qwen/Qwen-7B` and `meta-llama/Llama-3.1-8B-Instruct` are added.

Note that for models that has context window > 32768, we need to use `enable_chunked_prefill=False` and `max_model_len=32768`. Chunked prefill is automatically enabled in vllm when context window >32768.

@Siddhant-Ray Can we add corresponding document for the models we currently support?
@Shaoting-Feng Can we add tests for more models?

### CODE REVIEW

**Comment by Siddhant-Ray** [comment]
Will add documentation for other models by this weekend.

**Comment by Shaoting-Feng** [comment]
Will add tests for other models by this weekend.

File: examples/offline_inference/offline_inference.py [hunk 1]
```diff
@@ -7,6 +7,8 @@
 from transformers import AutoTokenizer
 
 model_name = "mistralai/Mistral-7B-Instruct-v0.2"
+#model_name = "Qwen/Qwen-7B"
+#model_name = "meta-llama/Llama-3.1-8B-Instruct"
```
Comment (KuntaiDu) [inline examples/offline_inference/offline_inference.py:11]: Would be great if you can remove these 2 line

**Review by KuntaiDu — APPROVED** [review]
LGTM!

**Comment by ApostaC** [comment]
As a side note, we still want to do some refactoring for the CacheGenConfig and have a "default configuration" for different models.

Just created a new ticket for this: #148

**Pull request merged**